src/common/utils/snapshots.ts

```diff
@@ -14,13 +14,13 @@ export async function fetchMasterSnapshotNumber(): Promise<number> {
 
 export function snapshotFromUrl(url: string): number {
   assert(url, 'Empty URL provided');
-  const match = /(\d+)\/?$/.exec(url);
+  const match = /(\d+)\/v2\/?$/.exec(url);
```

Comment:
I noticed that https://covid-projections-git-updating-to-new-api.covidactnow.vercel.app/internal/compare isn't working and I think it's because we end up getting the current snapshot URL from _master_ (see fetchMasterSnapshotNumber() above) which doesn't include /v2/...  I think the email alerts may do the same thing.  Should we make this regex accept either the old or new snapshot URLs?  Else, depending on how we deploy things we may have a temporary breakage?